chainntnfs/bitcoindnotify/bitcoind.go

```diff
@@ -69,6 +69,9 @@ type BitcoindNotifier struct {
 	// which the transaction could have confirmed within the chain.
 	confirmHintCache chainntnfs.ConfirmHintCache
 
+	// mempool gives the client access to transactions in the mempool.
+	mempool *chainntnfs.Mempool
```

Comment:
If made to be a distinct package/sub-system, then we can also add it to the `btcd` backend as well.

Reply:
cool this is no longer relevant